src/middlewared/middlewared/plugins/vm/vms.py

```diff
@@ -141,6 +143,9 @@ async def do_create(self, data):
 
         `hide_from_msr` is a boolean which when set will hide the KVM hypervisor from standard MSR based discovery and
         is useful to enable when doing GPU passthrough.
+
+        `hyperv_enlightenments` can be used to enable subset of predefined Hyper-V enlightenments for Windows guests. These
```

Comment:
Can we perhaps make it `Str('hyperv_enlightenments', null=True, enum=[None, 'WINDOWS']),` to allow more enlightenments presets for different types of OSes? @sonicaj

Reply:
I think this is a good start, we do have some other hypervisor features which other OSes can benefit from but that would be separate from hyperv changes.